src/isNotValidNumber.js

```diff
@@ -0,0 +1,24 @@
+import { complement } from 'ramda';
+import isValidNumber from './isValidNumber';
+
+/* eslint-disable max-len */
+/**
+ * Checks if value is not a valid `Number` object or is `NaN`,
+ * `POSITIVE_INFINITY` or `NEGATIVE_INFINITY`.
+ *
+ * @func isNotValidNumber
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/X.X.X|vX.X.X}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {Boolean}
+ * @example
+ *
+ * RA.isNotValidNumber(1); //=> false
+ * RA.isNotValidNumber(''); //=> true
+ */
+/* eslint-enable max-len */
```

Comment:
Drop this. Your lines length should not go over 100 chars

Reply:
Not sure what you mean.  What should I drop?

Reply:
```javascript
/* eslint-enable max-len */

Reply:
Ah. Sorry. Though you'd wrapped all your comment blocks in the enable/disable comments, but was just the one I cribbed from.